src/api/usersRouter.ts

```diff
@@ -19,25 +19,8 @@ import {
 
 export function createUsersRouter() {
   const usersRouter = express.Router();
-  const emailParser = z.object({
-    email: z.string().optional(),
-  });
-
-  usersRouter.get('/', async (req, res) => {
-    const { email } = emailParser.parse(req.query);
-    if (!email) {
-      const users = await User.query();
-      return res.status(200).json(users);
-    }
-    const userResult = await findUserByEmail(email);
-    if (userResult) return res.json(userResult);
-    return res.sendStatus(404);
-  });
-
-  usersRouter.post('/', async (req, res) => {
```

Comment:
This is an old endpoint - used to be used in account creation but now we do it by including the user in the post to the participants router.